### PR TITLE
Updated README file

### PR DESCRIPTION
**Pull request by rahu-eGov**

### CODE REVIEW

File: domain-services/ifix-department-entity-service/README.md [hunk 1]
```diff
@@ -1 +1,21 @@
...
+**parent:** It provides details about department parent.
+
+Root level department hierarchy will not contain any parent value and level value will be zero.\
+Level value incrementation rule:
+1. When parent id is having any value then we search parent in department hierarchy record for hierarchy level evaluation.
+2. Get level value from parent department hierarchy and increment current department hierarchy level value by one.
+
+
+## Department Entity
+It contains department entity information along with its hierarchy level and also attaches master department information (department id - like DWSS).
```
Comment (rushang7-eGov) [inline domain-services/ifix-department-entity-service/README.md:17]: We are referring to department's uuid not code.

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]
Also, add
1. First we need to define the hierarchy level top to bottom because it has parent's reference.
2. Only then we can start adding department entities, bottom-to-top because it has child's reference.

In the end, also mention:
The root hierarchy will have the label "Department", and the root entity will be the department itself.

File: domain-services/ifix-department-entity-service/README.md [hunk 2]
```diff
@@ -1 +1,21 @@
-# iFIX-Department-Entity-Service
\ No newline at end of file
+# iFIX-Department-Entity-Service
+
+## Department Hierarchy
+It defines hierarchy definition for department.
+
+**department id:** It is master department id like DWSS.\
```
[the remark this answers is not in the document]
Comment (rahu-eGov) [reply domain-services/ifix-department-entity-service/README.md]: done

File: domain-services/ifix-department-entity-service/README.md [hunk 3]
```diff
@@ -1 +1,21 @@
...
+**level:** It defines the depth of hierarchy of department level.\
+**parent:** It provides details about department parent.
```
[the remark this answers is not in the document]
Comment (rahu-eGov) [reply domain-services/ifix-department-entity-service/README.md]: done

File: domain-services/ifix-department-entity-service/README.md (hunk 1 above)
Comment (rahu-eGov) [reply domain-services/ifix-department-entity-service/README.md]: done

**Review by rushang7-eGov — APPROVED** [review]

**Pull request merged**